src/client/src/react/lib.test.js

```diff
@@ -110,4 +110,21 @@ describe("setsWon", () => {
     expect(setsPlayer1).toBe(0);
     expect(setsPlayer2).toBe(0);
   });
+
+  test.todo("test with more than one set");
+});
+
+describe("test isMatchFinished()", () => {
```

Comment:
in matchmaker/match -> getMatchWinner(match) this function is implemented as well